src/Sha512YaoGarbler.cc

```diff
@@ -1,14 +1,14 @@
 #include <cassert>
 #include <openssl/evp.h>
 #include "Exceptions.hh"
-#include "Utils.hh"
+#include "MathUtils.hh"
 #include "Sha512YaoGarbler.hh"
 
 // Output of SHA-512 is 128 characters long,
 // when represented in hexadecimal.
 const size_t SHA512_HEX_SIZE = EVP_MAX_MD_SIZE * 2;
-const std::string CHECK_HEX_STRING = "abcd";
```

Comment:
This, and its related changes, are not relevant to this PR. One can change them later in refactoring.